### PR TITLE
Add postgress nested with clause

### PR DESCRIPTION
**Pull request by MathewJoseph31**
Nested With Clauses, so that they can be of the form

WITH A as (  
WITH B AS ( 
WITH C AS (
.
.
)
)
)

 supported by latest DB versions, eg. postgres is not currently supported by the parser

### CODE REVIEW

**Comment by manticore-projects** [comment]
The `Nested With Items` implementation has been succeeded by PR #1754.

**Comment by wumpz** [comment]
So this can be closed, right?

**Comment by manticore-projects** [comment]
Please leave, i will try to salvage the other parts!

**Comment by manticore-projects** [comment]
Sorry, but this thing is too messy to salvage. I will close it but you can re-submit when there is any serious interest.

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt [hunk 1]
```diff
@@ -1723,6 +1737,9 @@ Expression RegularCondition():
 	| token=<OP_MINORTHANEQUALS> { result = new MinorThanEquals(token.image); }
 	| token=<OP_NOTEQUALSSTANDARD> { result = new NotEqualsTo(token.image); }
 	| token=<OP_NOTEQUALSBANG> { result = new NotEqualsTo(token.image); }
+	| token=<OP_DOUBLEAND> { result = new DoubleAnd(token.image); }
+	| token=<OP_CONTAINS> { result = new DoubleAnd(token.image); }
```
Comment (manticore-projects) [inline src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:1741]: Why create new `DoubleAnd` for the tokens `OP_CONTAINS` or `OP_CONTAINEDBY`

**Pull request closed without merging**